Extension/src/LanguageServer/configurations.ts

```diff
@@ -1598,7 +1598,7 @@ export class CppProperties {
             }
 
             // Get compiler path without arguments before checking if it exists
-            resolvedCompilerPath = compilerPathAndArgs.compilerPath;
+            resolvedCompilerPath = compilerPathAndArgs.compilerPath ?? undefined;
```

Comment:
FYI, `| null` could be added to the type of resolvedCompilerPath, but I'm not sure if that matters.

Reply:
Thanks for catching that.